mfxconfig.h.in

```diff
@@ -19,3 +19,10 @@
 // SOFTWARE.
 
 #cmakedefine MFX_ENABLE_KERNELS
+
+#cmakedefine MFX_ENABLE_MPEG2_VIDEO_DECODE
+#cmakedefine MFX_ENABLE_MPEG2_VIDEO_ENCODE
+
+#if defined(MFX_ENABLE_MPEG2_VIDEO_ENCODE) && defined(MFX_VA)
+#define MFX_ENABLE_MPEG2_VIDEO_ENCODE_HW
```

Comment:
We need to understand direction we are heading. I definitely don't like an idea to expose *_HW. Especially now when we don't have any SW fallback support. If we are sure it will not appear, then this definitely should be only * left and we can proceed to remove *_HW right away. @onabiull : how do you think?